dags.yaml

```diff
@@ -287,3 +287,17 @@ bqetl_marketing_fetch:
     retries: 2
     retry_delay: 30m
 
+bqetl_desktop_platform:
+  schedule_interval: 0 3 * * *
+  default_args:
+    owner: jklukas@mozilla.com
+    start_date: "2020-12-10"
```

Comment:
@jklukas , a quick follow up, I noticed that when I build a query to use the data from `moz-fx-data-shared-prod.telemetry_derived.accessibility_clients_v1` I only get data starting from December 09, 2020. I was hoping to get the data all the way back to March 01, 2017 (all other start dates in this patch). Was I supposed to have this date as the one from 2017 as well?